changelogs/fragments/better_winrm_putfile_error.yml

```
@@ -0,0 +1,2 @@
+bugfixes:
+  - Allow for better information when winrm put_file failes. fixes #70361
```

I don't think this actually fixes the referenced issue, it just potentially gets us more info about why it failed.

```suggestion
  - winrm - preserve winrm forensic data on put_file failures
```